src/concepts/components/properties.md

```diff
@@ -1,26 +1,551 @@
 ---
-description: Parent to child communication
+description: The component could expose property attributes to receive data from the parent html component
 ---
 
 # Properties
 
-Properties enable child and parent components to communicate with each other.
+It is a common need to receive data from the parent component using properties.
 
-## Derive macro
+The properties are defined in a dedicated Rust struct derivating `Properties` and `Clone`traits.
 
-Don't try to implement `Properties` yourself, derive it by using `#[derive(Properties)]` instead.
+> This struct is usually named `Props`
 
-{% hint style="info" %}
-Types for which you derive `Properties` must also implement `Clone`. This can be done by either using `#[derive(Properties, Clone)` or manually implementing `Clone` for your type.
-{% endhint %}
+The properties bag is attached to the state adding the field `props` in the component struct and defining the `Properties` `Type` in the component trait implementation.
```

Comment:
What is the properties bag?

Reply:
I used to speak about "bag" for a "container" (struct instance in that case) passed around methods. Inside this "container" there is key/values. 
... don't mind
I will remove it from the doc when I retrieve it.